libraries/ui-utils/src/main/kotlin/io/element/android/libraries/ui/utils/time/DurationExt.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.libraries.ui.utils.time
+
+import kotlin.time.Duration
+
+/**
+ * Format a duration as minutes:seconds.
+ *
+ * For example,
+ * - 0 seconds will be formatted as "0:00".
+ * - 65 seconds will be formatted as "1:05".
+ * - 2 hours will be formatted as "120:00".
+ * - negative 10 seconds will be formatted as "-0:10".
+ *
+ * @return the formatted duration.
+ */
+fun Duration.formatShort(): String {
```

Comment:
@bmarty Is there already amodule for such utilities?

Reply:
I thought about putting it in `libraries:core` but it felt a bit too low level for this extension. And I thought `libraries:ui-utils` could start to grow with some common Compose utilities too (e.g. `SoftKeyboardEffect` which I didn't find a proper home for).

Reply:
Could be in the `designsystem` module, as per thinks for [modifiers](https://github.com/vector-im/element-x-android/tree/develop/libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/modifiers).

But we can update this later.